### DISCUSSION TITLE
WSL Unable to Connect with Python's NiceGUI

### DISCUSSION THREAD

**Discussion by Nobunaji** [microsoft/WSL#9820] [Q&A]
I'm trying to learn the python library nicegui, i've managed to install it on my WSL2 and when i run my python code everything works but when i open the url in my windows browser it says 'Unable to connect'. 

Please help.

Thank You to whoever took time to read this post.

**Comment by ChrisJefferson** [comment]
I had the same problem, and found your comment.

When I load an example it says `NiceGUI ready to go on http://0.0.0.0:8080`, however this does not work.

However, `http://localhost:8080` does work (at least for the simplest example).
